index.html

```diff
@@ -129,7 +129,7 @@ <h2>Multi-Post <br>
     </section>
 
     <!-- About myself section -->
-    <!-- <section class="about_myself">
+     <section class="about_myself">
         <h2>About Myself</h2>
```

Comment:
- [ ] Please add some horizontal spacing in this section. We want it to look as close as possible to the Figma Template. It is essential to have a good design 😍 
You can achieve that by setting your **padding**